clientv3/watch.go

```diff
@@ -118,8 +118,8 @@ type watchGrpcStream struct {
 
 	// mu protects the streams map
 	mu sync.RWMutex
-	// streams holds all active watchers
-	streams map[int64]*watcherStream
+	// substreams holds all active watchers
```

Comment:
That's in `watcher.streams`, this tracks the watch id's that are in a single grpc stream (hence the rename from streams to substreams, using "streams" twice is super confusing). Will clarify the comments a bit.